index.html

```diff
@@ -1,38 +1,85 @@
     <link rel="stylesheet" href="css/index.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Cedarville+Cursive&display=swap" rel="stylesheet">
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cedarville+Cursive&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
 </head>
+
 <body>
+
     <h1>Joel DeLeon</h1>
-<section id="About">
-    <h2>About</h2>
-    <p>My Name is Joel DeLeon, and im learnig about programming with code the dream.</p>
-</section>
-
-<section id="Experience">
-<h2>Experience</h2>
-<ul>
-    <li>Basketball Coach</li>
-    <li>PCA</li>
-    <li>delivery driver</li>
-</ul>
-</section>
-
-<section id="Connect">
-    <h2>Connect</h2>
-
-<a href="https://www.youtube.com/@Theloredude">YouTube</a>
-<i class="fab fa-youtube"></i>
-</section>
+    <nav>
+        <ul>
+            <li><a href="#About">About</a></li>
+            <li><a href="#Experience">Experience</a></li>
+            <li><a href="#Connect"></a>Connect</li>
+            <li><a href="#Skills"></a>Skills</li>
+            <li><a href="#Projects"></a>Projects</li>
+        </ul>
+    </nav>
+    <section id="About">
+        <h2>About</h2>
+        <p>My Name is Joel DeLeon, and im learnig about programming with code the dream.</p>
+    </section>
+
+    <section id="Experience">
+        <h2>Experience</h2>
+        <ul>
+            <li>
+                <div class="job-header">
+                    <div class="job-title">Basketball Coach</div>
+                    <div class="job-dates">Dates</div>
+
+                </div>
+                <div class="job-description">Coach youth Basketball players in skills and shooting.</div>
+
+
+            </li>
+            <li>
+                <div class="job-header">
+                    <div class="job-title">PCA</div>
+                    <div class="job-dates">Dates</div>
+
+                </div>
+                <div class="job-description">Teach and take care of special needs.</div>
+
+
+            </li>
+            <li>
+                <div class="job-header">
+                    <div class="job-title">Delivery driver</div>
+                    <div class="job-dates">Dates</div>
+
+                </div>
+                <div class="job-description">Deliver packages, take picture of delivery, make sure records are up o
+                    date.</div>
+
+            </li>
+        </ul>
+    </section>
+
+    <section id="Connect">
+        <h2>Connect</h2>
+
+        <a href="https://www.youtube.com/@Theloredude">YouTube</a>
+        <i class="fab fa-youtube"></i>
+    </section>
```

Comment:
Since you only have one social media icon, you won't be able to see flexbox working even after you fix the CSS issues. Please add more items and try using CSS properties to better organize your layout. You could use something like this in the HTML:
```html
<section id="Connect">
    <h2>Connect</h2>
    <ul>
        <li>
            <a href="https://www.youtube.com/@Theloredude">YouTube</a>
            <i class="fab fa-youtube"></i>
        </li>
        <li>
            <a href="https://someotherlink.com">Another link</a>
            <i class="fab another-icon"></i>
        </li>
    </ul>
</section>
```
Then your CSS could use flexbox properties on the list.